db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
```

Comment:
@cben, can you give a quick example of a `labeled_resource_type`?  My eyes are still tricking me into thinking this has *something* to do with polymorphic relationships, even though I know it doesn't.